app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageController.scala

```diff
@@ -62,37 +69,104 @@ class MultipleDisposalsTriageController @Inject() (
+                        _.copy(numberOfProperties = Some(numberOfProperties)),
+                        _.copy(numberOfProperties = numberOfProperties)
+                      )
+                    ) -> routes.MultipleDisposalsTriageController.checkYourAnswers()
+                  else
+                    Right[MultipleDisposalsTriageAnswers, IncompleteSingleDisposalTriageAnswers](
+                      IncompleteSingleDisposalTriageAnswers.empty.copy(
+                        individualUserType         = answers.fold(_.individualUserType, c => Some(c.individualUserType)),
+                        hasConfirmedSingleDisposal = true
+                      )
+                    ) -> routes.SingleDisposalsTriageController.checkYourAnswers()
+
+                val newState = state.copy(newReturnTriageAnswers = newAnswersWithRedirectTo._1)
+
+                updateSession(sessionStore, request)(_.copy(journeyStatus = Some(newState))).map {
+                  case Left(e) =>
+                    logger.warn("Could not update session", e)
+                    errorHandler.errorResult()
+
+                  case Right(_) =>
+                    Redirect(newAnswersWithRedirectTo._2)
+                }
+
+              }
+
+            }
+          )
     }
   }
 
   def checkYourAnswers(): Action[AnyContent] = authenticatedActionWithSessionData.async { implicit request =>
     withMultipleDisposalTriageAnswers(request) {
       case (_, _, triageAnswers) =>
         triageAnswers match {
-          case IncompleteMultipleDisposalsAnswers(None) =>
+          case IncompleteMultipleDisposalsAnswers(None, _) =>
             Redirect(routes.InitialTriageQuestionsController.howManyProperties())
 
-          case IncompleteMultipleDisposalsAnswers(Some(_)) =>
+          case IncompleteMultipleDisposalsAnswers(Some(_), None) =>
             Redirect(routes.MultipleDisposalsTriageController.guidance())
 
+          case IncompleteMultipleDisposalsAnswers(_, Some(n)) =>
+            Ok(s"Got number of disposals $n")
+
           case c: CompleteMultipleDisposalsAnswers =>
             Ok(s"Got $c")
 
         }
-
     }
   }
 
   private def withMultipleDisposalTriageAnswers(request: RequestWithSessionData[_])(
-    f: (SessionData, Either[StartingNewDraftReturn, FillingOutReturn], MultipleDisposalsTriageAnswers) => Future[Result]
+    f: (SessionData, StartingNewDraftReturn, MultipleDisposalsTriageAnswers) => Future[Result]
   ): Future[Result] =
     request.sessionData.flatMap(s => s.journeyStatus.map(s -> _)) match {
       case Some((session, s @ StartingNewDraftReturn(_, _, _, Left(t)))) =>
-        f(session, Left(s), t)
+        f(session, s, t)
 
       case _ =>
         Redirect(uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.routes.StartController.start())
     }
 
 }
+object MultipleDisposalsTriageController {
+
+  val numberOfPropertiesForm: Form[Int] = Form(
+    mapping(
+      "numberOfProperties" -> of[Int]
```

Comment:
this should be `text` instead of `of[Int]` and then checking that
- the string is not empty
- the string can be converted to an Int 
- the Int is strictly greater than zero and strictly less than 1000
The error messages for those can be found in the ticket